cmd/manager/main.go

```diff
@@ -57,7 +57,7 @@ func main() {
 	var addonAgentImageName string
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":38080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":38081", "The address the probe endpoint binds to.")
-	flag.StringVar(&addonAgentImageName, "agent-image-name", "yue9944882/managed-serviceaccount-addon-agent:v0.0.0",
+	flag.StringVar(&addonAgentImageName, "agent-image-name", "open-cluster-maangement/managed-serviceaccount-addon-agent:v0.0.0",
```

Comment:
I think it should be quay.io/open-cluster-management?

Reply:
https://github.com/open-cluster-management-io/managed-serviceaccount/blob/741d9c369e6822c98591c91b9fe86e571d44a5e5/charts/managed-serviceaccount/templates/manager-deployment.yaml#L24

the default value doesnt take effect in helm installation tho

Reply:
oops... fix the typo